### DISCUSSION TITLE
Debug

### DISCUSSION THREAD

**Discussion by SzotsGabor** [paulolemus/blunders#4] [General]
Hi Paulo,

Will it play better or worse if I switch off the debug setting?

**Comment by paulolemus** [comment]
Hi Gabor,

The debug setting is just used for extra debug prints over UCI during a search currently. In a tight loop (such as a search benchmark) I noticed better performance with debug=false. In the context of a normal game it may perform only ever so slightly better than debug=true. Cheers!
